app/src/test/java/org/oppia/android/app/testing/options/AppLanguageFragmentTest.kt

```diff
@@ -83,6 +86,11 @@ private const val CHINESE = 3
 @Config(application = AppLanguageFragmentTest.TestApplication::class)
 class AppLanguageFragmentTest {
 
+  @get:Rule
+  var activityTestRule: ActivityTestRule<AppLanguageActivity> = ActivityTestRule(
```

Comment:
Use `val` instead of `var`

Reply:
Done.
I think you have mentioned me bymistake on a [different PR ](https://github.com/oppia/oppia-android/pull/2839#issuecomment-792609406)

Reply:
@abhinav12k Apologies. Just missed the last characters in user name.